.github/application.properties.ci

```diff
@@ -15,7 +15,7 @@ sentry.dsn=
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.url=jdbc:postgresql://localhost:5432/formplayer?prepareThreshold=0
 spring.datasource.username=postgres
-spring.datasource.password=
+spring.datasource.password=postgres
```

Comment:
Is this the recommended way to set secrets for automation tests (honestly not sure)?

Reply:
This isn't really a secret per se since it's defined in the build configuration file. I do plan to switch to using environment variables for these and dropping this file entirely but was leaving that for a future PR.

Actual secrets are set in the repo settings and accessible in the build configuration as part of the action context: https://docs.github.com/en/actions/security-guides/encrypted-secrets